mhkit/wave/io/wecsim.py

```diff
@@ -0,0 +1,422 @@
+import pandas as pd
+import numpy as np
+import scipy.io as sio
+
+def read_wecSim(file_name):
+    """
+    Loads the wecSim response class once 'output' has been saved to a *.mat 
+    structure. 
+    
+    NOTE: Python is unable to import MATLAB objects. 
+    MATLAB must be used to save the wecSim object as a structure. 
```

Comment:
These 3 lines say the same thing 3 different ways. I think we can be more succinct while still be clear. Something to this effect? 
```
    Loads the WEC-Sim response class.
    
    NOTE: MATLAB must be used to save the WEC-Sim object as a structure because Python is unable to import MATLAB objects.
```